cypress/integration/adminLogin.spec.js

```diff
@@ -0,0 +1,26 @@
+import '@testing-library/cypress/add-commands';
+
+describe('Administrator login', () => {
+  it('should login the initial administrator user from the home page', () => {
+    cy.fixture('logins/initialAdmin.json').as('admin');
```

Comment:
`.as(someString)` commands are aliases. This allows us to use the fixture via an aliased name. In the case of this fixture, we can now access it as `cy.get('@admin')`. Aliases are not just for fixture data though as you can see in the rest of the test below. For more info, please see the official Cypress documentation on [Variables and Aliases](https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Aliases)